tests/tests-scripts.php

```diff
@@ -101,6 +101,9 @@ public function test_register_styles_checkout_ssl() {
 		edd_register_styles();
 
 		$this->go_to( '/' );
+
+		$this->assertTrue( wp_style_is( 'edd-styles', 'registered' ) );
```

Comment:
I wasn't sure what this test was actually trying to accomplish. I just added this in to make it so at least one assertion was happening here. (Without assertions it was a "risky" test.)